### PR TITLE
Migrate InfoBanner to Chakra

### PR DESCRIPTION
**Pull request by ImMiguelP**
## Description

- Migrates `InfoBanner` component from `emotion` to `chakra-ui`

## Related Issue

#6374

### CODE REVIEW

File: src/components/InfoBanner.tsx [hunk 1]
```diff
@@ -75,17 +21,38 @@ const InfoBanner: React.FC<IProps> = ({
...
-      isWarning={isWarning}
-      shouldCenter={shouldCenter}
+    <Flex
+      alignItems="center"
+      p={6}
+      borderRadius={0.5}
+      maxW={shouldCenter ? "55rem" : "100%"}
+      color="black300"
+      bg={isWarning ? "warning" : "infoBanner"}
+      direction={{ sm: "column" }}
```
Comment (pettinarip) [inline src/components/InfoBanner.tsx:31]: This should be
```suggestion
      direction={{ base: "column", sm: "row" }}
```

File: src/components/InfoBanner.tsx [hunk 2]
```diff
@@ -75,17 +21,38 @@ const InfoBanner: React.FC<IProps> = ({
   ...props
 }) => {
   const banner = (
-    <Banner
-      className={className}
-      isWarning={isWarning}
-      shouldCenter={shouldCenter}
+    <Flex
+      alignItems="center"
```
Comment (pettinarip) [inline src/components/InfoBanner.tsx:25]:
```suggestion
      align="center"
```

File: src/components/InfoBanner.tsx [hunk 3]
```diff
@@ -75,17 +21,38 @@ const InfoBanner: React.FC<IProps> = ({
...
+      p={6}
+      borderRadius={0.5}
```
Comment (pettinarip) [inline src/components/InfoBanner.tsx:27]:
```suggestion
      borderRadius="sm"
```

File: src/components/InfoBanner.tsx [hunk 4]
```diff
@@ -75,17 +21,38 @@ const InfoBanner: React.FC<IProps> = ({
...
+      maxW={shouldCenter ? "55rem" : "100%"}
+      color="black300"
+      bg={isWarning ? "warning" : "infoBanner"}
+      direction={{ sm: "column" }}
       {...props}
```
Comment (pettinarip) [inline src/components/InfoBanner.tsx:32]: nice

File: src/components/InfoBanner.tsx [hunk 5]
```diff
@@ -75,17 +21,38 @@ const InfoBanner: React.FC<IProps> = ({
...
     >
-      {emoji && <StyledEmoji text={emoji} size={2} />}
-      <Content shouldSpaceBetween={shouldSpaceBetween}>{children}</Content>
-    </Banner>
+      {emoji && (
+        <Emoji
+          flexGrow="0"
+          flexShrink="0"
+          mr={{base : 6, sm: 0}}
```
Comment (pettinarip) [inline src/components/InfoBanner.tsx:38]:
```suggestion
          mr={{base : 0, sm: 6}}
```

File: src/components/InfoBanner.tsx [hunk 6]
```diff
@@ -75,17 +21,38 @@ const InfoBanner: React.FC<IProps> = ({
...
+          mb={{ sm: 2 }}
```
Comment (pettinarip) [inline src/components/InfoBanner.tsx:39]:
```suggestion
          mb={{ base: 2, sm: 0 }}
```

File: src/components/InfoBanner.tsx [hunk 7]
```diff
@@ -75,17 +21,38 @@ const InfoBanner: React.FC<IProps> = ({
...
+          alignSelf={{ sm: "flex-start" }}
```
Comment (pettinarip) [inline src/components/InfoBanner.tsx:40]:
```suggestion
          alignSelf={{ base: "flex-start", sm: "auto" }}
```

File: src/components/InfoBanner.tsx [hunk 8]
```diff
@@ -75,17 +21,38 @@ const InfoBanner: React.FC<IProps> = ({
...
+          text={emoji}
+          fontSize={"2em"}
+        />
+      )}
+      <Flex
+        display={{ base: shouldSpaceBetween ? "flex" : "block", sm: "block" }}
```
Comment (pettinarip) [inline src/components/InfoBanner.tsx:46]:
```suggestion
        display={{ base: "block", sm: shouldSpaceBetween ? "flex" : "block" }}
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Looking good @ImMiguelP left a few comments. Thanks for the PR

**Comment by ImMiguelP** [comment]
> Looking good @ImMiguelP left a few comments. Thanks for the PR

Thank you @pettinarip here goes the updated files with the correct changes

**Review by pettinarip — APPROVED** [review]
Awesome @ImMiguelP! Thanks for the PR!

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by pettinarip** [comment]
@all-contributors please add @ImMiguelP for code

**Pull request merged**